packages/application-shell/src/components/application-shell/redirect-to-login.tsx

```diff
@@ -0,0 +1,100 @@
+import type { ApplicationWindow } from '@commercetools-frontend/constants';
+import type { AuthorizeSessionState } from '../authenticated/types';
+
+import React from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useLocation } from 'react-router-dom';
+import {
+  joinPaths,
+  trimLeadingAndTrailingSlashes,
+} from '@commercetools-frontend/url-utils';
+import { LOGOUT_REASONS } from '@commercetools-frontend/constants';
+import { STORAGE_KEYS, OIDC_RESPONSE_TYPES } from '../../constants';
+import { buildOidcScope } from '../authenticated/helpers';
+import Redirector from '../redirector';
+
+declare let window: ApplicationWindow;
+
+const generateAndCacheNonceWithState = (state: AuthorizeSessionState) => {
+  const nonce = uuidv4();
+  // We store additional information within the given `nonce`
+  // to then retrieve it later when the IdP redirects back
+  // to our application. The URL will contain the `nonce` within
+  // the id_token and, once validated, we can retrieve and use
+  // the state object.
+  // https://auth0.com/docs/protocols/oauth2/oauth-state#how-to-use-the-parameter-to-restore-application-state
+  window.sessionStorage.setItem(
+    `${STORAGE_KEYS.NONCE}_${nonce}`,
+    JSON.stringify(state)
+  );
+  return nonce;
+};
+
+const RedirectToLogin = () => {
+  const location = useLocation();
+
+  if (window.app.__DEVELOPMENT__) {
```

Comment:
For local development, we send the construct the authorize request and redirect to the real login page.